test/.eslintrc.js

```diff
@@ -24,7 +24,8 @@ module.exports = {
         // here: https://issues.newrelic.com/browse/NEWRELIC-3321
         'node/no-unsupported-features/es-syntax': 'off',
         'node/no-unpublished-import': 'off',
-        'node/no-extraneous-import': 'off'
+        'node/no-extraneous-import': 'off',
+        'node/no-missing-import': 'off'
```

Comment:
I looked at the import statements and they all looked valid, so trying to figure out why we need to disable this rule?

Reply:
so I have this [PR](https://github.com/newrelic/eslint-config-newrelic/pull/11/files), which will remove there here. But this is failing because pg and you'll need in mongodb don't exist.  They get installed with the versioned runner. so any `import('pg')` fail because of this.

Reply:
Got it, we run linting on all files with our "normal" package.json deps, which doesn't have `pg` as part of the list, because pg only gets installed on the fly when running `npm run versioned`. Thank you for the clarification

Reply:
and may not fail for you running locally if you have pg installed in the versioned folder.